src/psyclone/psyir/nodes/node.py

```diff
@@ -891,6 +891,22 @@ def _insert_schedule(self, children=None, ast=None):
             sched.ast = ast
         return sched
 
+    def find_symbol_table(self):
+        '''
+        :returns: the symbol table attached to the nearest ancestor \
+            node (including self).
+        :rtype: :py:class:`psyclone.psyir.symbols.SymbolTable`
+
+        :raises InternalError: if no symbol table is found.
+
+        '''
+        current = self
+        while current and not hasattr(current, "symbol_table"):
+            current=current.parent
```

Comment:
pylint: missing whitespace.

Reply:
pylint also complains about `GenerationError` being re-imported at line 263. Although this isn't your fault would you mind removing it since we're here?

Reply:
Done and done.